framework/templates/ecl-templates-search/ecl-templates-search.twig

```diff
@@ -10,77 +10,128 @@
+                  {% endembed %}
+                {% endblock %}
+              {% endembed %}
+              {% embed '@ec-europa/ecl-forms-form-groups' %}
+                {% block form_group %}
+                  {% include '@ec-europa/ecl-forms-labels' with { 'label': 'File Format', 'for_attribute': 'search-option-file-format' } %}
+                  {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'file-format', 'id': 'search-option-file-format' } %}
+                    {% block options %}
+                      <option value="all">All formats</option>
+                      <option value="doc">Doc</option>
+                      <option value="pdf">PDF</option>
+                      <option value="web">Web</option>
+                    {% endblock %}
+                  {% endembed %}
+                {% endblock %}
+              {% endembed %}
+            {% endblock %}
+          {% endembed %}
+          {% embed '@ec-europa/ecl-forms-fieldsets' only %}
+            {% block fieldset %}
+              {% include '@ec-europa/ecl-forms-legends' with { content: 'Filter by topic', level: 1 } only %}
+              <p class="ecl-paragraph">Custom component</p>
+            {% endblock %}
+          {% endembed %}
+          <div class="ecl-u-d-lg-none ecl-u-mv-l">
+            {% include '@ec-europa/ecl-buttons' with {
+              'modifier': 'primary',
+              'isBlock': true,
+              'extraClass': 'ecl-u-mv-s ecl-u-mh-auto',
+              'label': 'Refine results',
+              'extraAttributes': [
+                { 'name': 'type', 'value': 'submit'}
+              ]
+            } %}
+            {% include '@ec-europa/ecl-buttons' with {
+              'label': 'Clear all',
+              'isBlock': true,
+              'extraClass': 'ecl-u-mv-s ecl-u-mh-auto',
+              'extraAttributes': [
+                { 'name': 'type', 'value': 'reset'}
+              ]
+            } %}
+          </div>
+        {% endblock %}
+      {% endembed %}
     </aside>
     <section class="ecl-col-lg-9">
-      <h2 class="ecl-heading ecl-heading--h2">Initiatives (65)</h2>
+      <h2 class="ecl-heading ecl-heading--h2 ecl-u-d-none ecl-u-d-lg-block">Initiatives (65)</h2>
       <h3 class="ecl-heading ecl-heading--h3">Showing results 11 to 20</h3>
-
-      <div class="ecl-u-d-flex">
-        {% include '@ec-europa/ecl-tags' with {
-          'modifier': 'facet-close',
-          'label': 'Source',
-          'tags': [{ 'value': 'DG Trade' }]
-        } %}
-        {% include '@ec-europa/ecl-tags' with {
-          'modifier': 'facet-close',
-          'label': 'Topics',
-          'tags': [{ 'value': 'Health, wellbeing & Consumer protection' }]
+      <div class="ecl-u-d-flex ecl-u-justify-content-between ecl-u-align-items-center ecl-u-flex-wrap">
+        <div class="ecl-u-d-flex ecl-u-flex-wrap ecl-u-mv-xxs"">
+          {% include '@ec-europa/ecl-tags' with {
+            'modifier': 'facet-close',
```

Comment:
This attribute is not rendered correctly, please check for the other occurrences as well.

![modifier_tag](https://user-images.githubusercontent.com/1923476/29620928-a0d750ce-881f-11e7-8f9d-10c7375e45ef.png)